renderer.go

```diff
@@ -229,3 +276,11 @@ func calcDrawSize(o Options) drawSize {
 		pieceOffset: int((gridSize - pieceSize) / 2),
 	}
 }
+
+func reverse(s string) string {
```

Comment:
@ZurabWeb Since this is only used internally to flip the string and both places are recommended to remove it, you can remove this as well (unless there is another use I'm missing).